### ISSUE TITLE
Fdroid update

### ISSUE THREAD

**Issue by AleixDev** [SoupeauCaillou/recursive-runner#5]
It seems that the game [is no longer in the F-Droid store][1]. Can you make a new tag from the latest commits so a new release can be build? 

(Related to #1)

[1]: https://f-droid.org/en/packages/#q=Runner

**Comment by bagage** [comment]
No, I'd rather ask for a [rebuild instead](https://gitlab.com/fdroid/fdroiddata/merge_requests/2612).

**Comment by AleixDev** [comment]
Okay perfect then! :+1:

**Comment by Jakarrrg** [comment]
It should be updated to work on modern devices. It's a shame. The aesthetics were outstanding and the gameplay fun, simple and with everlasting interest. All those years passed, and I still looked for it today to check if there was some update.
